types/account.go

```diff
@@ -3,97 +3,56 @@ package types
 import (
 	"bytes"
 	"encoding/json"
-	"fmt"
 
-	"gopkg.in/yaml.v2"
+	yaml "gopkg.in/yaml.v2"
 
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
-	"github.com/cosmos/cosmos-sdk/x/auth"
-	"github.com/cosmos/cosmos-sdk/x/auth/exported"
 	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
 
 	ethcmn "github.com/ethereum/go-ethereum/common"
 	ethcrypto "github.com/ethereum/go-ethereum/crypto"
 )
 
-var _ exported.Account = (*EthAccount)(nil)
-var _ exported.GenesisAccount = (*EthAccount)(nil)
-
-func init() {
-	authtypes.RegisterAccountTypeCodec(&EthAccount{}, EthAccountName)
-}
```

Comment:
do we not need to init and run `RegisterAccounTypeCodec()` anymore?

Reply:
no, we don't use amino